baseband/gsb/base.py

```diff
@@ -56,31 +56,38 @@ class GSBFileReader(VLBIFileBase):
     """Simple reader for GSB data files.
 
     Adds ``read_payload`` method to the basic VLBI binary file wrapper.
-    """
 
-    def read_payload(self, payloadsize, nchan=1, bps=4, complex_data=False):
+    Parameters
+    ----------
+    payloadsize : int
+        Number of bytes to read.
+    nchan : int
+        Number of channels in the data.  Default: 1.
+    bps : int
+        Number of bits per sample (or real/imaginary component).
+        Default: 4.
```

Comment:
```
bps : int, optional
    Bits per elementary sample; i.e. per real or imaginary component for
    complex data.  Default: 4.
```
Not that I'm in love with that description, but having everything the same will make it a cinch to replace if we feel like it.